Source/ButtonAContext.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cs505.group1.state;
```

Comment:
You need to create the same folder structure of your source folder to reflect your package structure.  Not having that means that the code doesn't compile based on how it is in GitHub.  For this make sure to move the files using "git mv" so that way it preserves the commit history of the file.

Reply:
Completed in  reorganize-folders  branch.